specs-go/config.go

```diff
@@ -364,7 +365,7 @@ type LinuxResources struct {
 	Pids *LinuxPids `json:"pids,omitempty"`
 	// BlockIO restriction configuration
 	BlockIO *LinuxBlockIO `json:"blockIO,omitempty"`
-	// Hugetlb limit (in bytes)
+	// Hugetlb limits (in bytes). Default to reservation limits if supported.
```

Comment:
Singular reads correctly to me. Similar to the files, `hugetlb.<hugepagesize>.limit_in_bytes` and `hugetlb.<hugepagesize>.rsvd.limit_in_bytes` of https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v1/hugetlb.html.

```suggestion
	// Hugetlb limit (in bytes). Default to reservation limit if supported.
```

Reply:
The field name is `HugepageLimits`, and it is a slice of limits, so I think the correct way to describe it is "limits" not "limit".

Reply:
Ah, yes. Agreed!